src/rax.c

```diff
@@ -206,6 +206,7 @@ rax *raxNew(void) {
     rax->numele = 0;
     rax->numnodes = 1;
     rax->head = raxNewNode(0,0);
+    rax->alloc = sizeof(rax) + rax_alloc_size(rax->head);
```

Comment:
`sizeof(rax)` here would be the size of the variable, but you meant the size of the struct.
either use `sizeof(*rax)` (like the allocation does), or `sizeof(struct rax)`

Reply:
p.s. considering that we're collecting the size **including** the internal fragmentation (i.e. using `malloc_size` rather than try to keep track of the amount we requested from the allocator), in theory this should be `malloc_size(rax)`.
in practice, `struct rax` has 32 bytes in 64 bit builds, and 24 bytes in 32 bit builds, so i think it is unlikely to have any internal fragmentation...

actually in 32 bit, with the default jemalloc library (used in debian distro), which has lg-quantum of 4, i think we won't have a 24 bytes bin, so allocating these 24 bytes will consume 32 bytes..

since we're using mallc_size all of the the place, no harm in having one more at "startup"